src/encoded/tests/data/master-inserts/static_section.json

```diff
@@ -958,7 +958,16 @@
     },
 
 
-
+    {
+        "name" : "basic-carousel.slide",
+        "uuid" : "442c8aa0-dc6c-43d7-814a-854af4600028",
+        "body" : "<BasicCarousel\r\n    slides={[\r\n\t\t\t{\r\n                'img': '\/static\/img\/Metadata_structure_slides\/Slide01.png',\r\n                'alt': 'Slide 1 alt text'\r\n            },\r\n            {\r\n                'img': '\/static\/img\/Metadata_structure_slides\/Slide02.png',\r\n                'alt': 'Slide 2 alt text'\r\n            },\r\n            {\r\n                'img': '\/static\/img\/Metadata_structure_slides\/Slide03.png',\r\n                'alt': 'Slide 3 alt text'\r\n            }\r\n\t]}\r\n    slideWidth={720}\r\n    slideHeight={540}\r\n    showSlideCount={true}\r\n    autoPlay={true}\r\n    key='random-basic-carousel-key'\r\n\/>",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary escape character: \/. (no-useless-escape)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4495549)

Reply:
Hm... I should figure out how to exclude .json files from ESLint maybe.. it might be a setting in Codacy dashboard if not an ESLint config setting.

Reply:
@alexkb0009 , updated the <BasicCarousel ...> of master-inserts/static_section.json. Will push it shortly. Possibly not fail in Codacy validation this time.